elastic/apm-server
# Measure Jaeger request metrics in shared gRPC server

## Motivation/summary

Add Jaeger monitoring map to the shared gRPC server's metrics interceptor, so Jaeger traffic sent to port 8200 is measured and does not lead to errors being logged.

## How to test these changes

1. Run apm-server with default config
2. Send Jaeger spans to port 8200
3. Check that there are no errors in the log

## Related issues

Closes #5176
